management-api-server/src/test/java/com/datastax/mgmtapi/BaseDockerIntegrationTest.java

```diff
@@ -59,7 +59,7 @@ protected void failed(Throwable e, Description description) {
           System.err.flush();
 
           if (null != docker) {
-            int numberOfLines = 100;
+            int numberOfLines = 1000;
```

Comment:
thought:
I upped this here as it's the number of lines of the Cassandra logfile dumped out during a failed IT test. Sometimes the error is more than 100 lines from the end of the logfile. Since this is only for tests, I think it's ok to up this.

Reply:
I'm OK with this too :)